test/e2e/l7networkpolicy_test.go

```diff
@@ -332,4 +348,64 @@ func testL7NetworkPolicyTLS(t *testing.T, data *TestData) {
 
 	probeL7NetworkPolicyTLS(t, data, clientPodName, "apis.google.com", false)
 	probeL7NetworkPolicyTLS(t, data, clientPodName, "www.facebook.com", true)
+
+	data.crdClient.CrdV1beta1().NetworkPolicies(data.testNamespace).Delete(context.TODO(), policyAllowSNIFacebook, metav1.DeleteOptions{})
+}
+
+func testL7NetworkPolicyLogging(t *testing.T, data *TestData) {
+	l7LoggingNode := nodeName(0)
+	l7LogDir := logDir + "l7engine/eve-" + time.Now().Format(time.DateOnly) + ".json"
+
+	clientPodName := "test-l7-logging-client-selected"
+	clientPodLabels := map[string]string{"test-l7-logging-e2e": "client"}
+	cmd := []string{"bash", "-c", "sleep 3600"}
```

Comment:
You don't need an explicit sleep command (you don't need `WithCommand`) as the agnhost image will pause indefinitely by default.

Reply:
It makes sense, though I don't understand as why we need it [there](https://github.com/antrea-io/antrea/blob/572790d207ef8da205efb12b2c497809fdf96a2a/test/e2e/framework.go#L2855) but not here?

Reply:
It should not be needed there either. I usually remove these occurrences as I find them.